guides/doc-Upgrading_and_Updating/topics/upgrading_process_overview.adoc

```diff
@@ -101,6 +101,18 @@ You must migrate a self-registered {Project} to the Red Hat Content Delivery Net
 To migrate a self-registered {Project} to the CDN, see {UpgradingDocURL}Upgrading_Server_upgrade-guide[Upgrading {ProjectName}] in the _Upgrading and Updating {ProjectName}_ guide.
 endif::[]
 
+ifdef::satellite[]
+.Considerations for Upgrades of {Project} to Future Versions
+
+When you upgrade to {Project} 6.10, you are required to upgrade to Pulp 3, which can be time consuming due to Pulp content migration.
+For {SmartProxies}, you can choose to re-deploy new 6.10 {SmartProxies} rather than to upgrade.
+
+When {Project} 7.0 is released, you will be required to upgrade the operating system from RHEL 7 to RHEL 8 on your {ProjectServer}s and {SmartProxies}.
+You can upgrade the operating system in-place or through a cloning process that includes Pulp content migration.
+
+However, if you are planning to avoid the upgrade from Pulp 2 to Pulp 3 and deploy a new {Project} 6.10 infrastructure due to the Pulp 3 changes instead, you may want to wait for 7.0 to deploy with RHEL 8 directly.
```

Comment:
This is the key point, that we want to provide users with plenty of advance notice so that they can plan the path that works best for them. The prior points in this section are more like context to inform users why this may make sense.

For that reason, should this sentence be bolded, or use some kind of caution sign or other symbol to draw attention to it?

Reply:
@wbclark I was thinking the same, I'll make it into a Note.